setup.py

```diff
@@ -14,7 +14,7 @@ def install_data_files_hack():
 
 install_data_files_hack()
 
-requires = ['six', 'flask', 'Werkzeug>=0.15.0', 'pygments', 'dulwich>=0.19.3', 'httpauth', 'humanize']
+requires = ['six', 'flask', 'Werkzeug>=0.15.0', 'pygments', 'dulwich>=0.19.3', 'httpauth', 'humanize', 'dulwich<0.20;python_version<"3.0"']
```

Comment:
Having both `dulwich>=0.19.3` and `dulwich<0.20;python_version<"3.0"` looks strange to me.

Just saying, feel free to ignore my observation ...

Reply:
You're right, these weren't quite correct - adjusted now (and travis agrees).

We do need two entries, because there are different constraints for the different Python versions.